src/patterns/components/navigation/angular/info/secondary.hbs

```diff
@@ -0,0 +1,63 @@
+<p class="sprk-u-mbm">See below for available customization options:</p>
+<table class="drizzle-c-Table drizzle-c-Table--wide sprk-u-mbm">
+  <colgroup>
+    <col class="sprk-u-Width-40">
+    <col>
+  </colgroup>
+  <thead>
+  <tr>
+    <th>Component</th>
+    <th>Input</th>
+    <th>Type</th>
+    <th>Description</th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td>sprk-secondary-navigation</td>
+    <td class="sprk-u-FontWeight--bold">additionalClasses</td>
+    <td>string</td>
+    <td>Expects a space-separated list of class names to append to the class attribute
+      of the list element inside of sprk-secondary-navigation.</td>
+  </tr>
+  <tr>
+    <td>sprk-secondary-navigation</td>
+    <td class="sprk-u-FontWeight--bold">additionalNavClasses</td>
+    <td>string</td>
+    <td>Expects a space-separated list of class names to append to the class attribute
+      of the nav element inside of sprk-secondary-navigation.</td>
+  </tr>
+  <tr>
+    <td>sprk-secondary-navigation-item</td>
+    <td class="sprk-u-FontWeight--bold">additionalClasses</td>
+    <td>string</td>
+    <td>Expects a space-separated list of class names to append to the class attribute
+      of sprk-secondary-navigation-item.</td>
+  </tr>
+  <tr>
+    <td>sprk-secondary-navigation-item</td>
+    <td class="sprk-u-FontWeight--bold">analyticsString</td>
+    <td>string</td>
+    <td>The text that is assigned to the data-analytics attribute of the spark-secondary-navigation-item.</td>
+  </tr>
+  <tr>
+    <td>sprk-secondary-navigation-item</td>
+    <td class="sprk-u-FontWeight--bold">href</td>
+    <td>string</td>
+    <td>The text that is assigned to the href attribute of the link.</td>
+  </tr>
+  <tr>
+    <td>sprk-secondary-navigation-item</td>
+    <td class="sprk-u-FontWeight--bold">spacing</td>
+    <td>string</td>
+    <td>Determines which predefined space size is set between the secondary navigation items.
+      The available values are: 'tiny', 'small', 'large', and 'huge'.</td>
```

Comment:
Is medium an option?

Reply:
its the default, i should probably mention that